### PR TITLE
Adding extra information to user profile

### PR DESCRIPTION
**Pull request by akuleshov7**
### What's done:
- Polishing profile view: adding extra info, like free text, button to settings and other
- Fixing alignments on Top Rating view

image

### CODE REVIEW

**Review by Cheshiriks — APPROVED** [review]

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/HasErrorModal.kt [hunk 1]
```diff
@@ -25,7 +25,7 @@ import web.html.ButtonType
  */
 @Suppress("MAGIC_NUMBER")
 val ringLoader = ringLoader(jso {
-    this.size = 70
+    this.size = 80
```
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/HasErrorModal.kt:28]: What does `80` mean? 

I mean, unit-wise?

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/AvatarRenderers.kt [hunk 2]
```diff
@@ -75,26 +75,39 @@ fun ChildrenBuilder.renderAvatar(
  * @param classes
  * @param link
  * @param styleBuilder
+ * @param isHorizontal if the avatar shoud be on the same line with text
  */
 fun ChildrenBuilder.renderUserAvatarWithName(
     userInfo: UserInfo,
     classes: String = "",
     link: String? = null,
+    isHorizontal: Boolean = false,
```
Comment (sanyavertolet) [inline save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/AvatarRenderers.kt:84]: we surely needed it

**Review by sanyavertolet — APPROVED** [review]
lgtm

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/AvatarRenderers.kt (hunk 2 above)
Comment (akuleshov7) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/basic/AvatarRenderers.kt:84]: yeah, but it's also a quick hack

File: save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/HasErrorModal.kt (hunk 1 above)
Comment (akuleshov7) [reply save-frontend/src/main/kotlin/com/saveourtool/save/frontend/components/HasErrorModal.kt:28]: I am think, that it's px )

File: save-frontend/src/main/resources/img/company.png
Comment (nulls) [inline save-frontend/src/main/resources/img/company.png:1]: 😸

**Pull request merged**